x/nexus/keeper/chain.go

```diff
@@ -90,6 +90,15 @@ func (k Keeper) ActivateChain(ctx sdk.Context, chain exported.Chain) {
 	k.setChainState(ctx, chainState)
 }
 
+// DeactivateChain deactivates the given chain
+func (k Keeper) DeactivateChain(ctx sdk.Context, chain exported.Chain) {
+	chainState, _ := k.getChainState(ctx, chain)
+	chainState.Chain = chain
+	chainState.Activated = false
```

Comment:
This chain state struct has other arguments, what if there is no pre-existing chain state in the keeper?

Reply:
I think it will just get an empty chain state if there is no pre-existing state

Reply:
sure, but are we sure its ok to adopt the fields of such empty struct?